common/src/main/java/com/microsoft/identity/common/internal/cache/UnnamedClassThatDelegatesToTheActualCache.java

```diff
@@ -0,0 +1,88 @@
+package com.microsoft.identity.common.internal.cache;
+
+import android.content.Context;
+
+import com.microsoft.identity.common.internal.dto.AccessToken;
+import com.microsoft.identity.common.internal.dto.Account;
+import com.microsoft.identity.common.internal.providers.oauth2.AuthorizationRequest;
+import com.microsoft.identity.common.internal.providers.oauth2.OAuth2Strategy;
+import com.microsoft.identity.common.internal.providers.oauth2.OAuth2TokenCache;
+import com.microsoft.identity.common.internal.providers.oauth2.RefreshToken;
+import com.microsoft.identity.common.internal.providers.oauth2.TokenResponse;
+
+import java.util.List;
+
+public class UnnamedClassThatDelegatesToTheActualCache
```

Comment:
@shoatman Because this isn't _really_ the cache we should think about what this thing is called...